### PR TITLE
Added german translation

### PR DESCRIPTION
**Pull request by marcelschreiner**
Hi Fabrizio

Thanks for creating this awesome Spotify module. Using it has been a blast. 
In this pull request I added a german translation.

Greetings from Switzerland
Marcel

### CODE REVIEW

**Comment by Fabrizz** [comment]
Thanks @marcelschreiner!

**Comment by Fabrizz** [comment]
Hey @marcelschreiner sorry for bothering you, is: `"Spielen in einem anderen Raum."` the correct translation for `"Playing on another room."`? Thanks

**Comment by marcelschreiner** [comment]
Hey @Fabrizz no worries 😄 
It depends...

If you want to perform the action `"Play what is currently playing in another room"` then the correct translation is `"In einem anderen Raum wiedergeben"`

If you want to say `"The currently playing music is playing in an other room"` then the translation `"Die Musik wird in einem anderen Raum wiedergegeben"` would be correct.

p.s. in german the term `"Musik spielen"` is mostly used if you are playing an instrument and `"Musik wiedergeben"` is used if a speaker is playing music. But your sentence is still correct but not commonly used.

I hope this helps 😇

**Comment by marcelschreiner** [comment]
@Fabrizz a short addition

I used the translation `"Es wird nichts abgespielt"` in the proposed changes of this pull request.
If you use one of the suggested translation from my previous reply,  `"Es wird nichts wiedergegeben"` would be more fitting.

File: translations/de.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "DISPLAY_PRIVATE": "Private Spotify-Sitzung",
+    "PRODUCT_WARNING": "Das aktuelle Konto hat kein Spotify Premium. Dieses Modul erfordert ein Premium-Konto!",
+    "CONNECTION_WARNING": "Es gab einen Verbindungsfehler. Verlangsamung des Abfragezyklus von api.spotify/...",
+    "CONNECTION_ERROR": "Verbindungsfehler bestehen weiterhin. Weitere Verlangsamung des Abfragezyklus. Alte Player-Daten werden ausgeblendet.",
+    "VIBRANT_NOT_LOADED": "(vendor) Vibrant wurde nicht korrekt geladen.",
+    "USER_CSS_ERROR": "Es gab einen Fehler beim Extrahieren von CSS-Informationen. (Die Dauer von Animationen/Übergängen sollte in \"ms\" sein)",
+    "STABLISHING_CONNECTION": "Verbindung zu Spotify wird hergestellt...",
+    "NOTHING_PLAYING": "Es wird nichts abgespielt",
```
Comment (marcelschreiner) [inline translations/de.json:9]: "NOTHING_PLAYING": "Es wird nichts wiedergegeben",

File: translations/de.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "SPOTIFY_HEADER": "MMM-OnSpotify",
+    "DEBUG_COLORS": "Debug-Paletten werden angezeigt, weil <showDebugPalette> aktiviert ist",
+    "SPOTIFYCODE_EXPERIMENTAL": "<spotifyCodeExperimentalShow> ist aktiviert. Die API zum Generieren von Codes ist nicht öffentlich und kann unerwartet ändern. (https://github.com/Fabrizz/MMM-OnSpotify#other)",
+    "RESUME": "Modul fortsetzen",
+    "SUSPEND": "Modul anhalten",
+    "LIVELYRICS_NOTICE": "MMM-LiveLyrics gefunden! - Lies die MMM-LL Logs für mehr Infos. (https://github.com/Fabrizz/MMM-LiveLyrics)",
+    "CSSOVERRIDE_NOTICE": "Du überschreibst die Standard-MM2-CSS-Variablen. Überprüfe sie auf Rechtschreibfehler und Kontrastprobleme.",
+    "CSSOVERRIDE_MALFORMED": "Es gab einen Fehler beim Parsen \"experimentalCSSOverridesForMM2\" Arrays. Entferne deinen Eintrag oder überprüfe ihn auf Fehler."
```
Comment (marcelschreiner) [inline translations/de.json:17]: "CSSOVERRIDE_MALFORMED": "Es gab einen Fehler beim Parsen **_des_** \"experimentalCSSOverridesForMM2\" Arrays. Entferne deinen Eintrag oder überprüfe ihn auf Fehler."

I have forgotten a word, sorry 🙈

**Comment by Fabrizz** [comment]
Thanks a lot!. Will change them in the next release. The problem with "Die Musik wird in einem anderen Raum wiedergegeben" is that it will overflow outside the module. I had the same problem with the spanish translation that ended up being something like "Nothing playing here." its not the best, but for an optional feature of an optional feature...

**Pull request merged**